core/src/main/java/com/zextras/carbonio/user_management/services/UserService.java

```diff
@@ -45,29 +48,54 @@ private UserInfo createUserInfo(GetAccountInfoResponse accountInfo) {
     return userInfo;
   }
 
+  public Response getUsers(List<String> userIds, String token) {
+    return Response.ok().entity(
+      userIds.stream().distinct().map(userId -> {
+        System.out.println("Requested: " + userId);
+        UserInfo userInfo = cacheManager.getUserByIdCache().getIfPresent(userId);
+
+        if (userInfo == null) {
+          try {
+            GetAccountInfoResponse accountInfo = SoapClient
+              .newClient()
+              .setAuthToken(token)
+              .getAccountInfoById(userId);
+
+            userInfo = createUserInfo(accountInfo);
+            cacheManager.getUserByIdCache().put(userId, userInfo);
+            cacheManager.getUserByEmailCache().put(userInfo.getEmail(), userInfo);
+          } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
If I remember correctly e.printStackTrace() doesn't write to the journal because it uses System.err and not System.out. Check it and if so change the code to print in out